README.md

```diff
@@ -1,84 +1,85 @@
-cd /vagrant
-npm rebuild node-sass
-```
-
-The second is to rebuild the application.  First you need to cancel the app in the screen with `screen -x` then CTRL-C.  Then you can bring the app back up with one of the above commands or in another screen session with `screen -dmS build bash -c 'cd /vagrant; ng serve'`.
+# Welcome to Planet
+[![Maintainability](https://api.codeclimate.com/v1/badges/028682cc4cd969b05280/maintainability)](https://codeclimate.com/github/open-learning-exchange/planet/maintainability)
+[![Build Status](https://travis-ci.org/open-learning-exchange/planet.svg?branch=master)](https://travis-ci.org/open-learning-exchange/planet)
+
+## What's Planet
+
+Planet is a web-application that includes everything needed to create a platform for ~~organization~~ community to manage their learning content and learner collaboration which has 3 main components, community, nation, and earth.
+
+Understanding the 3 main components is key to understanding Planet. Planet divides it installed instance as community, nation, or earth. Organization can install Planet as community, nation, or earth.
+
+## Community
+
+Community is the smallest entity of Planet member, at it root, every installation of Planet is a community, but we call the installation (or node) as community if it doesn't have another community report to it. A community belong to nation and can exchange contents. Think of it as a subscriber in publisher-subscriber model.
+
+## Nation
+
+Nation is a an entity which has at least one community belongs to it and as a nation it can publish content which can be consumed by the communities. In its relationship with communities, think of it as a publisher in pub-sub model. The recommended setup is one country has one nation which all the communities in that country consumes its content.
+
+In Planet, nation has a unique role where it not only act as a publisher for community, but it is a subscribe of Earth where in our framework there is one Earth which managed by [Open Learning Exchange][OLE] where it backed the initiative (But you can be your own Earth too, it is an open source project anyway).
+
+## Earth
+
+Similar as a nation to community, earth act as a publisher - perhaps the upstream publisher for all the content the community might consumes. It is exist as a way to exchange contents between nation.
+
+## Technologies
+
+We use Angular for our front-end and CouchDB as our back-end and database. And for the deployment we highly support Docker, but an old-school deployment in bare-metal or VM is also supported. We also support Raspberry-Pi deployment and in fact we have a lot of communities running in this little machine.
+
+## Getting Started
+
+1. Install CouchDB 2.2.0
+
+    1.1. OSX
+
+        $ brew install couchdb
+
+    1.2. Ubuntu
+    
+    Follow the instruction in the official page [here][COUCHDB_INSTALL].
+
+    1.3. Docker (**Suggested!**)
+
+        $ docker run -p 5984:5984 -d --name my-couchdb treehouses/couchdb:2.2.0
```

Comment:
App defaults to db on port 2200, so we should use that.  Also we should put the suggested choice first on the list.